content/tests/components/pages/OverviewPage.test.tsx

```diff
@@ -100,4 +100,65 @@ describe("OverviewPage", () => {
 
     expect(NormandyAPI.prototype.disableRecipe).toBeCalled();
   });
+
+  it("should display need to pause recipes", async () => {
+    const recipes = recipeFactory.buildMany([{}, {}, {}]);
```

Comment:
If you don't care to specify any overrides, you can use the even simpler `const recipes = recipeFactory.buildCount(3);`